src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -200,7 +200,7 @@ export default Vue.extend({
           }, 200)
         }
 
-        $(document).on('keydown', this.keyboardShortcutHandler)
+        $(document).on('keyup', this.keyboardShortcutHandler)
```

Comment:
This can't be `keyup` as then the spacebar will scroll down the page before the video toggles between play / pause, so we need to keep this as `keydown`.